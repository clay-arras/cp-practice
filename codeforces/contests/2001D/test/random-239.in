1
22
20 21 11 7 1 21 19 25 8 9 9 25 20 18 16 7 7 13 10 25 18 6
