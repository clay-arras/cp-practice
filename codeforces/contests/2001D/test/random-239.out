14
21 11 7 1 19 8 9 20 18 16 13 10 25 6
